modules/sdk-coin-eth/test/unit/transferBuilder.ts

```diff
@@ -85,30 +73,6 @@ describe('Eth send multi sig builder', function () {
       should.equal(result, testData.SEND_TERC_DATA);
     });
 
-    it('erc20 transfer should succeed', async () => {
-      const builder = new TransferBuilder()
-        .coin('tcusd')
-        .expirationTime(1590078260)
-        .amount(amount)
-        .to(toAddress)
-        .contractSequenceId(0)
-        .key(key);
-      const result = builder.signAndBuild();
-      should.equal(result, testData.SEND_TOKEN_SEQUENCE_ZERO_DATA);
-    });
-
-    it('erc20 transfer with amount 0 should succeed', async () => {
-      const builder = new TransferBuilder()
-        .coin('tcusd')
-        .expirationTime(1590078260)
-        .amount('0')
-        .to(toAddress)
-        .contractSequenceId(2)
-        .key(key);
-      const result = builder.signAndBuild();
-      should.equal(result, testData.SEND_TOKEN_AMOUNT_ZERO_DATA);
-    });
-
```

Comment:
These tests have been moved to sdk-coin-celo module because the token used here is a celo token

Reply:
Nice!!!